sdk_container/src/third_party/coreos-overlay/sys-boot/shim-signed/shim-signed-15.8.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 The Flatcar Maintainers.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Signed UEFI Shim loader"
+HOMEPAGE="https://github.com/rhboot/shim"
+SRC_URI="https://mirror.release.flatcar-linux.net/coreos/shimx64.efi.signed"
```

Comment:
We need package version that the signed shim was built from to be part of the url somehow.